src/remote/util.py

```diff
@@ -67,7 +67,7 @@ def rsync(
     """
 
     logger.info("Sync files from %s to %s", src, dst)
-    args = ["rsync", "-rlpmchz", "--copy-unsafe-links", "-e", "ssh -q", "--force"]
```

Comment:
just a note for future: 
--copy-unsafe-links has its own set of problems...
ivy.xml files for example don't work when you pull build artifacts back. 

Need to think about a better strategy for symlinks are not portable.